scripts/download-fastq.sh

```diff
@@ -40,3 +43,10 @@ fi
 # Count the lines in the R2 file
 echo "The number of lines in $FASTQ_R2 is:"
 gunzip -c ${FASTQ_DEST}/${FASTQ_R2} | wc -l
+
+fastp \
+    --in1 $FASTQ_DEST/$FASTQ_R1 \
+    --in2 $FASTQ_DEST/$FASTQ_R2 \
+    --out1 $TRIMMED_DIR/$FASTQ_R1 \
+    --out2 $TRIMMED_DIR/$FASTQ_R2 \
+    --html "${REPORTS_DIR}/${STUDY_ID}_report.html"
```

Comment:
Let's put the JSON file into the `reports` directory also:
```suggestion
    --html "${REPORTS_DIR}/${STUDY_ID}_report.html" \
    --json "${REPORTS_DIR}/${STUDY_ID}_report.json"
```